common/src/keyword.rs

```diff
@@ -34,6 +34,9 @@ pub enum Primitive {
   I64,
   U32,
   U64,
+  Int,
+  Float,
+  Number,
```

Comment:
Why is this it's own keywords? Is it for type inference and if so wouldn't it make more sense with something like `usize` and `isize`?

Reply:
Or if it's not a keyword why not make it a new enum which is called something else but essentially extends this enum?